WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Payments Menu/InPersonPaymentsMenu.swift

```diff
@@ -7,124 +9,141 @@ struct InPersonPaymentsMenu: View {
-                                    title: Localization.manageCardReader)
-                    }
-                    .disabled(viewModel.shouldDisableManageCardReaders)
+                        NavigationLink {
+                            PaymentSettingsFlowPresentingView(viewModelsAndViews: viewModel.manageCardReadersViewModelsAndViews)
+                        } label: {
+                            PaymentsRow(image: Image(uiImage: .creditCardIcon),
+                                        title: Localization.manageCardReader)
+                        }
+                        .disabled(viewModel.shouldDisableManageCardReaders)
 
-                    NavigationLink {
-                        CardReaderManualsView()
-                    } label: {
-                        PaymentsRow(image: Image(uiImage: .cardReaderManualIcon),
-                                    title: Localization.cardReaderManuals)
+                        NavigationLink {
+                            CardReaderManualsView()
+                        } label: {
+                            PaymentsRow(image: Image(uiImage: .cardReaderManualIcon),
+                                        title: Localization.cardReaderManuals)
+                        }
+                    } header: {
+                        Text(Localization.cardReaderSectionTitle.uppercased())
+                    } footer: {
+                        InPersonPaymentsLearnMore(viewModel: .inPersonPayments(source: .paymentsMenu),
+                                                  showInfoIcon: false)
+                        .customOpenURL(binding: $viewModel.safariSheetURL)
                     }
-                } header: {
-                    Text(Localization.cardReaderSectionTitle)
-                } footer: {
-                    InPersonPaymentsLearnMore(viewModel: .inPersonPayments(source: .paymentsMenu),
-                                              showInfoIcon: false)
-                    .customOpenURL(binding: $viewModel.safariSheetURL)
-                }
-                .renderedIf(viewModel.shouldShowCardReaderSection)
-
-                Section {
-                    NavigationLink(isActive: $showingManagePaymentGateways) {
-                        InPersonPaymentsSelectPluginView(selectedPlugin: nil) { plugin in
-                            viewModel.dependencies.onboardingUseCase.clearPluginSelection()
-                            viewModel.dependencies.onboardingUseCase.selectPlugin(plugin)
-                            showingManagePaymentGateways = false
+                    .renderedIf(viewModel.shouldShowCardReaderSection)
+
+                    ScrollViewSection {
+                        NavigationLink(isActive: $showingManagePaymentGateways) {
+                            InPersonPaymentsSelectPluginView(selectedPlugin: nil) { plugin in
+                                viewModel.dependencies.onboardingUseCase.clearPluginSelection()
```

Comment:
At first glance it seems like the view knows too much about the view model structure and its collaborators. It makes it tightly coupled and more difficult to change in the future ([Law of Demeter](https://blog.testdouble.com/posts/2022-06-15-law-of-demeter/#:~:text=The%20Law%20of%20Demeter%2C%20simply,coding%20when%20they%20coined%20it.)) On the other hand, it might be that this approach simplifies things ... Did you consider that?

Reply:
So, this is nothing new it's just indented/moved into a new type of section.

However, perhaps this action should be a function defined in the view model, rather than handled by the view.

The `dependencies` struct makes it look a bit worse than it is too, as that could be flattened to show that `onboardingUseCase` is owned by the viewModel. Still, it does involve a close coupling of the view and view model. I'll tidy it up.

Reply:
Done in 8c6ce01